tests/parquet_test.py

```diff
@@ -120,3 +120,35 @@ def test_get_datasets(self):
 
         for f in glob.glob('pq_test*'):
             os.remove(f)
+
+    @pytest.mark.optional_parquet
+    def test_against_standard_files(self):
+        datadir = 'resources/parquet-testing'
+        filenames = ['alltypes_plain.parquet',
+                     'alltypes_plain.snappy.parquet',
+                     'delta_byte_array.parquet']
+        columns1 = ['id',
+                    'bool_col',
+                    'tinyint_col',
+                    'smallint_col',
+                    'int_col',
+                    'bigint_col',
+                    'float_col',
+                    'double_col',
+                    'date_string_col',
+                    'string_col']
```

Comment:
As far as the exclusion of the timestamp column goes, I think we should actually include that column since it is deprecated and we technically can read the column, we just won't be getting correct results.

I looked into this for a bit and it seems difficult to differentiate between a timestamp of int96 and int64 programmatically, and I think part of that is because the int96 timestamps have been deprecated.

If we are concerned about being able to detect int96 vs int64 timestamps and skip the int96 timestamp columns, I could invest more time in that, but I wonder the likelihood of that scenario given that int96 timestamps are deprecated.

Is the skipping of int96 timestamp columns essential?